.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": false,
```

Comment:
Semicolons should be true

Reply:
as on discord I think semicolons should be false

Reply:
👌🏼